### ISSUE TITLE
IDISP003 should not warn when discarding

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#206]
```cs
public void M()
{
    _ = File.OpenRead(string.Empty);
    _ = File.OpenRead(string.Empty);
}
```
